scripts/normalize_jest_coverage.js

```diff
@@ -0,0 +1,27 @@
+const libCoverage = require("istanbul-lib-coverage")
+const { createReporter } = require("istanbul-api")
+const fs = require("fs")
+const path = require("path")
+
+const normalizeJestCoverage = result => {
+  Object.entries(result).forEach(([k, v]) => {
+    if (v.data) result[k] = v.data
+  })
+
+  return result
+}
+
+const map = libCoverage.createCoverageMap()
+const jestCoverage = require("../.nyc_output/coverage-final.json")
+
+map.merge(normalizeJestCoverage(jestCoverage))
+
+const reporter = createReporter()
+reporter.addAll(["json"])
+reporter.write(map)
+
+fs.copyFile(
+  "./coverage/coverage-final.json",
+  "./.nyc_output/coverage-final.json",
+  error => error && console.log(error)
```

Comment:
It might be worthwhile to keep logging consistent and verbose, so that we're able to track down precicely where the error occurred. Something like [seen here](https://github.com/artsy/force/blob/master/scripts/uploadToS3.js#L44)